index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Middle</title>
+</head>
+<body>
+    <section class="work">
+        <div class="work-title">
+            <h1 class="work-title-h1">My Recent Works</h1>
+            
+        </div>
+        <div class="work-items">
+            <hr>
+        </div>
+        <div class="work-items">
+            <img src="img/ImgPlaceholder.png" alt="">
+            <h2>
+                Multi-Post Stories
+            </h2>
+            <p class="work-items-p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
```

Comment:
- [ ] Kindly add the list of technologies and link to the project just like the Figma design.
![Captura de pantalla_2021-09-18_02-21-03](https://user-images.githubusercontent.com/53111464/133880266-aa5c85a9-610f-4353-89aa-950314f0513f.png)